tests/scp/test_scp_copy.py

```diff
@@ -0,0 +1,78 @@
+import os
+import pytest
+import time
+from tests.common.helpers.assertions import pytest_assert
+import logging
+import pwd
+
+pytestmark = [
+    pytest.mark.disable_loganalyzer,
+    pytest.mark.topology("any"),
+    pytest.mark.device_type("vs"),
+]
+
+SCP_PORT = 22
+TEST_FILE_NAME = "test_file.bin"
+TEST_FILE_2_NAME = "test_file_2.bin"
+
+@pytest.fixture
+def setup_teardown(duthosts, rand_one_dut_hostname, ptfhost):
+    duthost = duthosts[rand_one_dut_hostname]
+
+    # Copies script to DUT
+    duthost.copy(src="scp/perform_scp.py", dest="/home/admin/perform_scp.py")
+
+    yield
+
+    files_to_remove = ["./{}".format(TEST_FILE_NAME), "/home/admin/perform_scp.py"]
+    for file in files_to_remove:
+        duthost.file(path=file, state="absent")
+
+    files_to_remove_2 = ["./{}".format(TEST_FILE_NAME),"./{}".format(TEST_FILE_2_NAME)]
+    for file in files_to_remove_2:
+        ptfhost.file(path=file, state="absent")
+
+def test_scp_copy(duthosts, rand_one_dut_hostname, ptfhost, setup_teardown):
+    duthost = duthosts[rand_one_dut_hostname]
+    ptf_ip = ptfhost.mgmt_ip
+
+    duthost.copy(src="scp/perform_scp.py", dest="/home/admin/perform_scp.py")
+
+    # Generate the file from /dev/urandom
+    ptfhost.command(("dd if=/dev/urandom of=./{} count=1 bs=1000000000 iflag=fullblock".format(TEST_FILE_NAME)))
```

Comment:
Please define file size in a variable for easy change and consistent size checking if needed.

Reply:
Halved the size and also used a separate variable.